pandas/core/indexes/datetimes.py

```diff
@@ -819,6 +819,25 @@ def slice_indexer(self, start=None, end=None, step=None, kind=None):
 
     # --------------------------------------------------------------------
 
+    def _convert_listlike_indexer(self, key):
+        if not isinstance(key, list):
+            # There are no slices, so we can dispatch back
+            return super()._convert_listlike_indexer(key)
+
+        new_indexer = []
+        positions = list(range(len(self)))
+        try:
+            for k in key:
+                # Convert slices to list of integers
+                indexer = positions[self.get_loc(k)]
```

Comment:
can you use .get_indexer() here? (and still then iterate over the results)

Reply:
We discussed not supporting this. Has this changed?

Reply:
yah i think we want to close this